CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
Would have helped to put here 1.21.6; now I need to do that change myself. In future please do so.

Reply:
Okay!